test/memcache-helper.js

```diff
@@ -37,6 +36,9 @@ var config = {
   }
 }
 
+const spawn = require('child_process').spawnSync
```

Comment:
@deeptibaghel ah I see so we would need to use `spawn` here. I Think the tests hang because the process is left running after the tests are done in `ban_tests.js`

Reply:
`spawnSync` just blocks the process and doesn't continue because the `memcached` just sits there